website/profile/utils.py

```diff
@@ -107,6 +111,11 @@ def serialize_contributors(contribs, node, **kwargs):
         for contrib in contribs
     ]
 
+def serialize_visible_contributors(node):
+    # This is optimized when node has .include('contributor__user__guids')
+    return [
+        serialize_user(c, node) for c in node.contributor_set.all() if c.visible
```

Comment:
This could be a little faster, the difference will depend on the size of the list and the number that are or are not visible. `[serialize_user(c, node) for c in node.contributor_set.filter(visible=True)]`